### PR TITLE
Use pushBackUnique in Tags cache

### PR DESCRIPTION
**Pull request by jonpas**
**When merged this pull request will:**
- Title
- Add maintainers
- Use `configProperties` with inheritance instead of `configClasses`
- Invert a condition to be more readable

### CODE REVIEW

File: addons/tagging/functions/fnc_compileConfigTags.sqf [hunk 1]
```diff
@@ -46,8 +46,6 @@
 
     if (!_failure) then {
         GVAR(cachedTags) pushBack [_class, _displayName, _requiredItem, _textures, _icon];
-        if !(_requiredItem in GVAR(cachedRequiredItems)) then {
-            GVAR(cachedRequiredItems) pushBack _requiredItem;
-        };
+        GVAR(cachedRequiredItems) pushBackUnique _requiredItem;
     };
-} forEach ("true" configClasses (configFile >> "ACE_Tags"));
+} forEach (configProperties [configFile >> "ACE_Tags"]);
```
Comment (commy2) [inline addons/tagging/functions/fnc_compileConfigTags.sqf]: Will also include non-class entries, which would error out if you do e.g. `_x >> "displayName"`
I guess that's fine though as the class is not supposed to have such entries.

Comment (jonpas) [inline addons/tagging/functions/fnc_compileConfigTags.sqf]: Doesn't `configClasses` do the same? Return both entries and sub-classes?

Comment (commy2) [inline addons/tagging/functions/fnc_compileConfigTags.sqf]: No, only classes. That's why it's called `configCLASSES`.

Equivalent would be:
`configProperties [configFile >> "ACE_Tags", "isClass _x"];`

Comment (jonpas) [inline addons/tagging/functions/fnc_compileConfigTags.sqf]: Ah, I see, I'll change it back then to be safe.

**Pull request merged**